package.json

```diff
@@ -5,14 +5,16 @@
   "main": "index.js",
   "dependencies": {
     "acorn": "^5.7.1",
-    "node": "^10.6.0",
+    "meow": "^5.0.0",
+    "node": "^10.8.0",
```

Comment:
While we're here -- let's remove this dependency, I think it's unneeded.

Reply:
done